packages/native/types/index.d.ts

```diff
@@ -13,6 +13,9 @@ import {
   ReactNativeStyleType
 } from './base'
 
+import * as classComponent from '../src/classComponent'
```

Comment:
Since this module is shipping its source and the definition are stripped after compile, I think this should be ok? The key idea here is to keep the new work on this dry by just altering the js export of `react-native`.